uk.co.compucorp.civicrm.hrcontactactionsmenu/hrcontactactionsmenu.php

```diff
@@ -67,8 +67,12 @@ function hrcontactactionsmenu_civicrm_pageRun(&$page) {
     CRM_Core_Resources::singleton()->addStyleFile($extName, 'css/contactactions.css');
     CRM_Core_Resources::singleton()->addScriptFile($extName, 'js/contactactions.js');
 
-    $templatePath = CRM_Core_Resources::singleton()->getPath($extName,'templates/CRM/HRContactActionsMenu/Page/Inline/Actions.tpl');
-    $page->assign('alternativeActionsTemplate', $templatePath);
+    CRM_Core_Region::instance('contact-page-inline-actions')->update('default', array(
```

Comment:
I read the commit message, but maybe you could expand a bit on what's changed here in the PR description as I'm not so sure what it is.

Also we can use the `[]` array syntax in the above line

Reply:
Good point. I completely forgot to mention that in the PR description, but this is now fixed

Reply:
Thanks, that clears it up